<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="menuPage">
     <widget class="QGraphicsView" name="graphicsViewMenu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>601</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="levelOneButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kristen ITC</family>
        <pointsize>16</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Level One</string>
      </property>
     </widget>
     <widget class="QPushButton" name="levelTwoButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kristen ITC</family>
        <pointsize>16</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Level Two</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>450</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kristen ITC</family>
        <pointsize>16</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLevelOne">
     <widget class="QGraphicsView" name="graphicsViewLevelOne">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLevelTwo">
     <widget class="QGraphicsView" name="graphicsViewLevelTwo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageWin">
     <widget class="QGraphicsView" name="graphicsViewWin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="backToMenuButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLose">
     <widget class="QGraphicsView" name="graphicsViewLose">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="backToMenuButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
